pinot-controller/src/main/resources/app/components/Homepage/Operations/RebalanceServer/RebalanceServerOptions.ts

```diff
@@ -0,0 +1,145 @@
+export type RebalanceServerOption = {
+    name: string;
+    label: string;
+    type: "BOOL" | "INTEGER" | "SELECT";
+    description: string;
+    defaultValue: string | boolean | number;
+    isAdvancedConfig: boolean;
+    isStatsGatheringConfig: boolean;
+    hasBreakingChange: boolean;
+    allowedValues?: string[];
+}
+
+export const rebalanceServerOptions: RebalanceServerOption[] = [
+    {
+        "name": "dryRun",
+        "defaultValue": false,
+        "label": "Dry Run",
+        "type": "BOOL",
+        "description": "If enabled, rebalance will not run but expected changes that will occur will be returned",
+        "isAdvancedConfig": false,
+        "isStatsGatheringConfig": true,
+        "hasBreakingChange": false
+    },
+    {
+        "name": "preChecks",
+        "defaultValue": false,
+        "type": "BOOL",
+        "label": "Pre-Checks",
+        "description": "If enabled, will perform some pre-checks to ensure rebalance is safe, must enable dryRun to enable this",
```

Comment:
@somandal you can now configure what to show in the `UI` directly from this config file

Reply:
that's the way to go!

Reply:
can we make type: "BOOL" ... as enum ?